### PR TITLE
[SD-3048] Normalise the case for anpa category in vocabulary and ingest

### PR DESCRIPTION
**Pull request by marwoodandrew**
Publish filtering on anpa category was not working due to the case sensitive comparison. Ingest routing on anpa category was also broken along with genre.

### CODE REVIEW

File: server/apps/publish/publish_filters/filter_condition_tests.py [hunk 1]
```diff
@@ -353,6 +376,19 @@ def test_does_match_with_genre_filter(self):
...
+        f = FilterConditionService()
+        doc = {'field': 'anpa_category', 'operator': 'in', 'value': 'a,i'}
+        self.assertFalse(f.does_match(doc, self.articles[0]))
+        self.assertFalse(f.does_match(doc, self.articles[1]))
+        self.assertFalse(f.does_match(doc, self.articles[2]))
+        self.assertFalse(f.does_match(doc, self.articles[3]))
+        self.assertFalse(f.does_match(doc, self.articles[4]))
+        self.assertFalse(f.does_match(doc, self.articles[5]))
+        self.assertFalse(f.does_match(doc, self.articles[6]))
+        self.assertFalse(f.does_match(doc, self.articles[7]))
```
Comment (plamut) [inline server/apps/publish/publish_filters/filter_condition_tests.py]: Maybe just check the first `N-1` articles in a loop, but it's a minor thing.

**Comment by plamut** [comment]
:+1:

File: server/superdesk/io/commands/update_ingest.py [hunk 2]
```diff
@@ -221,6 +228,8 @@ def process_anpa_category(item, provider):
                     if anpa_category['is_active'] is True \
                             and item_category['qcode'].lower() == anpa_category['qcode'].lower():
```
Comment (sivakuna-aap) [inline server/superdesk/io/commands/update_ingest.py]: we don't need extra lower() for `anpa_category['qcode']` as it's already lower in case.

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Pull request merged**